.github/workflows/ci.yml

```
@@ -167,14 +168,16 @@ jobs:
           # - "3.9"
           # - "3.10"
           # - "3.11"
-          - "3.12"
+          - "3.12"  # Comment out when 3.13 is final
+          - "3.13"
         group: [1, 2]
 
     steps:
       - uses: actions/checkout@v4
       - uses: actions/setup-python@v5
         with:
           python-version: ${{ matrix.python }}
+          allow-prereleases: true # Temporarily required for Python 3.13
```

It's fine to leave this here after 3.13.0 final is out, the action  will install the full releases just the same. Compare the Ubuntu/Mac one. 

```suggestion
          allow-prereleases: true
```